src/HeaderCell.tsx

```diff
@@ -38,6 +39,14 @@ export const resizeHandleClassname = css`
   }
 `;
 
+const draggingClassname = css`
+  opacity: 0.5;
```

Comment:
This also changes the opacity of the border. Wonder if we should fix that

Reply:
Should we change the background color only instead?
Or maybe we can decrease the contrast with a css filter.